.github/workflows/linux_openvino_ci.yml

```diff
@@ -0,0 +1,45 @@
+name: Linux OpenVINO CI
+
+on:
+  push:
+    branches: [ main, 'rel-*' ]
+  pull_request:
+    branches: [ main, 'rel-*' ]
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+  packages: write # Needed if the reusable workflow pushes images
+  attestations: write # Optional: for artifact attestations if enabled
+  id-token: write # Optional: may be needed for OIDC authentication (e.g., ACR)
+
+jobs:
+  build_test_openvino:
+    name: Build and Test OpenVINO EP (AlamLinux8, Py3.12)
+    # Use the reusable workflow as the other Linux CI pipelines
+    uses: ./.github/workflows/reusable_linux_build.yml
+    with:
+      pool_name: "onnxruntime-github-Ubuntu2204-AMD-CPU"
+      build_config: Release
+      # Architecture: OpenVino only supports Intel X64
+      architecture: x64
+      dockerfile_path: tools/ci_build/github/linux/docker/inference/x86_64/python/openvino/Dockerfile
+      docker_image_repo: onnxruntimeopenvino
+
+      execution_providers: 'openvino'
+
+      extra_build_flags: '--use_openvino CPU --enable_generic_interface --build_shared_lib'
```

Comment:
The Windows OpenVINO CI Pipeline doesn't use the option `-enable_generic_interface`, while we are using the option here. Is this intentional? Should we consider updating to the same build flags for consistency?

Reply:
Currently the pipeline has it. So I keep it there.